event_model/documents/generate/type_wrapper.py

```diff
@@ -3,21 +3,39 @@
 """
 
 
+import pkgutil
 from dataclasses import dataclass
 
-try:
-    import pydantic
+pydantic_path = pkgutil.find_loader("pydantic")
+pydantic_version = None
+if pydantic_path:
+    import pydantic  # type: ignore
 
+    pydantic_version = pydantic.__version__
+
+if pydantic_version and pydantic_version < "2.0":
```

Comment:
I think this is probably fine for the short term but could bite us in the future e.g.
```python
>>> "10.0">"2.0"
False
```

Reply:
Hopefully we'll convert to pydantic 2 relatively soon, but I just fixed this just in case.